examples/auth0/pages/advanced/ssr-profile.js

```diff
@@ -20,7 +20,7 @@ function Profile({ user }) {
   )
 }
 
-Profile.getInitialProps = async ({ req, res }) => {
+export async function getServerSideProps({ req, res }) {
```

Comment:
In this function, remove anything that's not for the server, because `getServerSideProps` will only execute in the server, meaning anything that doesn't match `typeof window === 'undefined'`

Reply:
lol, I did that before, I forgot to leave it in the last commit